core/src/filed/filed.cc

```diff
@@ -66,6 +67,9 @@ static void usage()
         "        -g <group>  run as group <group>\n"
         "        -k          keep readall capabilities\n"
         "        -m          print kaboom output (for debugging)\n"
+#if !defined(HAVE_WIN32)
+        "        -p <file>   use <file> as destination when creating pidfile\n"
```

Comment:
```suggestion
        "        -p <full/path/to/pidfile>   use given path and filename when creating pidfile\n"
```

Reply:
nobody documents it like that. Compare `virtlockd --help`, `tuned --help`, `teamd --help` or `man rsyslogd`.